README.md

```diff
@@ -1 +1,24 @@
-# reverse-proxy
+# Using website-local-proxy
+
+This is a reverse proxy to handle the port-forwarding of the various locally-hosted Keyman sites. Assuming the corresponding sites already already running, the proxy can be started with the command-line:
+```bash
+build.sh build start
+```
+
+If you want the host to use a port other than 80, set the environment variable `PROXY_PORT` to the port number.
```

Comment:
This needs to filter through to all the docker hosted sites so that they rewrite their `<a href` URLs to the right port.

Also, we should probably call it `KEYMAN_COM_PROXY_PORT` to avoid generic env vars.

Reply:
I renamed to `KEYMAN_COM_PROXY_PORT`, but I'm still unclear how this gets used.  Could you elaborate more on
> This needs to filter through to all the docker hosted sites so that they rewrite their <a href URLs to the right port.

Reply:
TODO for those repos:
- [ ] http://api.keyman.com.localhost
- [ ] http://help.keyman.com.localhost
- [ ] http://keyman.com.localhost
- [ ] http://keymanweb.com.localhost
- [ ] http://s.keyman.com.localhost